analyses/cell-type-wilms-tumor-06/notebook/07_combined_annotation_across_samples_exploration.Rmd

```diff
@@ -113,23 +103,17 @@ sample_metadata_file <- file.path(repository_base, "data", "current", "SCPCP0000
 metadata <- read.table(sample_metadata_file, sep = "\t", header = TRUE)
 ```
 
-We extracted from the pre-processed and labeled `Seurat` object from `results/{sample_id}/06_infercnv_HMM-i3_{sample_id}_reference-{reference}.rds`.
-the following information per cell:
-
-- the predicted compartment in `fetal_kidney_predicted.compartment` and `fetal_kidney_predicted.compartment.score`
-
-- the predicted cell type in `fetal_kidney_predicted.cell_type` and `fetal_kidney_predicted.cell_type.score`
-
+We extracted from the pre-processed and labeled `Seurat` object from `results/{sample_id}/06_infercnv_HMM-i3_{sample_id}_reference-{reference}.rds`. the following information per cell:
 
-- the sample identifier in `sample_id`
+-   the predicted compartment in `fetal_kidney_predicted.compartment` and `fetal_kidney_predicted.compartment.score`
 
+-   the predicted cell type in `fetal_kidney_predicted.cell_type` and `fetal_kidney_predicted.cell_type.score`
 
-- the CNV `{dupli|loss}` estimation per chromosome {i} and arm `{p|q}`: 
-`has_{dupli|loss}_chr{i}{p|q}`, in 1 to 22
+-   the sample identifier in `sample_id`
 
+-   the CNV `{dupli|loss}` estimation per chromosome {i} and arm `{p|q}`: `has_{dupli|loss}_chr{i}{p|q}`, in 1 to 22
```

Comment:
I'm not sure what the last clause is here, I assume that it is supposed to say `i` in 1-22?
```suggestion
-   the CNV `{dupli|loss}` estimation per chromosome `{i}` and arm `{p|q}`: `has_{dupli|loss}_chr{i}{p|q}`, in 1 to 22
```